cypress/e2e/signIn.cy.js

```diff
@@ -28,7 +28,31 @@ describe('Sign In page', () => {
     homePage.assertHeaderContainUsername(user.username);
   });
 
-  it('should not provide an ability to log in with wrong credentials', () => {
+  it('should not provide an ability to log in with wrong email', () => {
+    signInPage.visit();
+    cy.register(user.email, user.username, user.password);
```

Comment:
You are registering a new user inside a test case that is meant to verify the scenario of logging in with a wrong email. This is confusing and the registration of the user should be done in a before or beforeEach block so that it's clear that this is setup for the test cases.

Reply:
done